libr/include/r_util/r_assert.h

```diff
@@ -0,0 +1,117 @@
+#ifndef R_ASSERT_H
+#define R_ASSERT_H
+
+#include "r_log.h"
+
+R_API void r_assert_log(RLogLevel level, const char *fmt, ...);
+
+#if defined (__GNUC__) && defined (__cplusplus)
+#define R_FUNCTION     ((const char*) (__PRETTY_FUNCTION__))
```

Comment:
i would remove the spaces after R_FUNCTION and before (

Reply:
ok